### ISSUE TITLE
near cli can't connect to testnet and mainnet

### ISSUE THREAD

**Issue by fospring** [near/near-cli#961]
***Describe the bug***
 when I input shell command of:
 `near keys yongchun.testnet`

***To Reproduce***
 1\. input command on shell:
 `near keys yongchun.testnet`
 2\. See error
 `Error: Exceeded 10 attempts for [https://rpc.testnet.near.org](https://rpc.testnet.near.org).`

***Expected behavior***
 return account's keys info

***Screenshots***
 !<span class="error">[image]</span>([https://user-images.githubusercontent.com/19633691/163384296-1398f8a6-de74-444c-8f30-ae9eb742c563.png](https://user-images.githubusercontent.com/19633691/163384296-1398f8a6-de74-444c-8f30-ae9eb742c563.png))

***Desktop (please complete the following information):***

*   OS: <span class="error">[MacOS M1]</span>
*   Version <span class="error">[11.5.1 ]</span>

***Additional context***
 near version is 3.2.0
 node version is v14.18.1

**Comment by fospring** [comment]
I use proxy by following command still fail:
```
export http_proxy="http://127.0.0.1:7890"
export https_proxy="http://127.0.0.1:7890"
```

**Comment by vKolerts** [comment]
please add support of proxy by env

**Comment by fospring** [comment]
> support of proxy by env

@vKolerts How should I do, can you please tell me more detail about it?

**Comment by fospring** [comment]
> > support of proxy by env
> 
> @vKolerts How should I do, can you please tell me more detail about it?

The proxy has make effects on `curl` but has not effect on `near` cli.

**Comment by fospring** [comment]
@vKolerts I resolve this issue according to this [article](https://issueantenna.com/repo/near/near-cli/issues/940)

**Comment by xiangxn** [comment]
All proxy problems can be solved in this way #1067
